commons/utils/jira_utils.py

```diff
@@ -328,7 +328,7 @@ def update_execution_details(self, test_run_id: str, test_id: str,
         Add comment to the mentioned jira id.
         """
         try:
-            url = "https://jts.seagate.com/rest/raven/1.0/testrun/{}/comment".format(test_run_id)
+            url = f"https://jts.seagate.com/rest/raven/1.0/api/testrun/{test_run_id}/comment"
```

Comment:
When did you notice that Jira API URI has changed ?

Reply:
@kachhwahadivya noticed the comments are not being updated for executed Test recently. So, I checked the URI, and updated it.